docs/UserGuide.md

```diff
@@ -191,6 +191,25 @@ characters or otherwise. For example, if "John Tan" exists in your contacts:
   * E.g: `add n/Hàn yǔ Pīn yīn p/98765432 e/hypy@example.com a/John street, block 123, #01-01` fails.
 
 
+***Names in FriendFolio***
+
+* Names are unique in FriendFolio, therefore people with the same name must be differentiated either with extra
+characters or otherwise. For example, if "John Tan" exists in your contacts:
+  * E.g: `add n/John Tan p/98765432 e/johnT@example.com a/John street, block 123, #01-01` fails.
+  * E.g: `add n/John Tan from SoC p/98765432 e/johnT@example.com a/John street, block 123, #01-01` succeeds.
+* Note that duplicate name detection is **case-sensitive**, therefore:
+  * E.g: `add n/john tan p/98765432 e/johnT@example.com a/John street, block 123, #01-01` also succeeds.
+* Names are alphanumeric only.
+  * E.g: `add n/Hàn yǔ Pīn yīn p/98765432 e/hypy@example.com a/John street, block 123, #01-01` fails.
+
+<div markdown="span" class="alert alert-primary">:motorway: **Future changes:**
+
+***Making Phone Number and Email Address Unique [Coming Soon]***
+
+FriendFolio is looking to make either the person's phone number and email address the unique identifier in future, to 
```

Comment:
I think the phrasing "either phone number and email" is a bit awkward. How about "FriendFolio is looking to make the person's phone number and email the unique identifiers in the future. This change aims to prevent multiple individuals from sharing the same email or phone number within the system while allowing multiple individuals with the same name to exist."